website/docs/cloud-docs/workspaces/create.mdx

```diff
@@ -0,0 +1,145 @@
+      Providers](/terraform/cloud-docs/vcs) for more details.
+   1. If you choose the **GitHub App** provider, choose an organization and repository when prompted. The list only displays the first 100 repositories from your VCS provider. If your repository is missing from the list, enter the repository ID in the text field .
+   1. Refer to the following topics for information about configuring workspaces settings in the **Advanced options** screen:
+      - [Terraform Working Directory][workdir]
+      - [Automatic Run Triggering][trigger]
+      - [VCS branch][branch]
+      - [Include submodules on clone][submodules]
+1. Specify a name for the workspace. VCS workflow workspaces default to the name of the repository. The name must be unique within the organization and can include letters, numbers, hyphens, and underscores. Refer to [Workspace naming](#workspace-naming) for additional information.
+1. Add an optional description for the workspace. The description appears at the top of the workspace in the HCP Terraform UI.
+1. Click **+ Add tag** and define an optional key-value pair. Tags help you organize workspaces. 
+
+   Workspaces inherit parent project tags. Reserved keys appear in the **Tag key** field as suggestions. Refer to [Create and manage reserved tags](/terraform/cloud-docs/users-teams-organizations/organizations#create-and-manage-reserved-tags) for additional information.
+
+    Refer to [Workspace tags](#define-tags) for character restrictions, inheritance, and other information.
+
+1. Add any additional tags.
+1. Click **Create workspace** to finish.
+
+For CLI or API-driven workflow, the system opens the new workspace overview. For version control workspaces, the **Configure Terraform variables** page appears.
+
+## Define workspace tags
+
+You can define key-value pairs to help you organize your workspaces and use them to sort by cost center, for example, to help you track resource consumption. You can define tags while [creating the workspace](#create-a-workspace) or create and attach them to an existing workspace.
+
+The tag management interfaces only support tags stored as key-value pairs. If your organization contains existing single-string tags, you must use the workspace API to manage the tags. Refer to [Workspaces API](/terraform/cloud-docs/api-docs/workspaces) for additional information.
+
+You cannot create tags for a workspace using the CLI.
+
+Complete the following steps to define workspace tags:
+
+1. Open your workspace.
+1. Click either the count link for the **Tags** label or **Manage Tags** in the **Tags** card on the right-sidebar to open the **Manage workspace tags** drawer.
+1. Click **+Add tag** to define a new key-value pair. The following rules apply to tag keys and values:
+   1. Tags must be one or more characters.
+   1. Tags have a 255 character limit.
+   1. Tabs can include letters, numbers, colons, hyphens, and underscores.
+   1. Tag values are optional.
```

Comment:
since this doesn't require numbered list, this will make it easier to parse. also typo (tabs -> tags)

```suggestion
   - Tags must be one or more characters.
   - Tags have a 255 character limit.
   - Tags can include letters, numbers, colons, hyphens, and underscores.
   - Tag values are optional.
```
<img width="722" alt="image" src="https://github.com/user-attachments/assets/00b6e471-144e-4ffc-a6c1-f393df0e9f6f">

Reply:
Thanks. I need to switch back to VS Code, this JetBrains IDE does all kinds of weird auto-formatting that I don't immediately see.